src/app/handlers/counting.handler.ts

```diff
@@ -63,6 +72,8 @@ export class CountingHandler extends Handler {
       return true;
     }
 
+    prevMessage.content = this._replaceWordsWithNumbers(prevMessage.content);
```

Comment:
Setting the content of a message is kinda weird. I'd prefer just making a `prevNumber` variable or something like that